taxcalc/functions.py

```diff
@@ -121,7 +121,7 @@ def CapGains(e23250, e22250, e23660, _sep, _feided, FEI_ec_c,
 def SSBenefits(SSIND, MARS, e02500, _ymod, e02400, SS_thd50, SS_thd85,
                SS_percentage1, SS_percentage2):
 
-    if SSIND != 0 or MARS == 3 or MARS == 6:
+    if SSIND == 2 or MARS == 3 or MARS == 6:
```

Comment:
I guess we are happy with the change at this moment. Thanks for your
comment.

On Thu, Oct 1, 2015 at 3:54 PM, feenberg <notifications@github.com> wrote:

> In taxcalc/functions.py
> <https://github.com/OpenSourcePolicyCenter/Tax-Calculator/pull/370#discussion_r40958429>
> :
>
> > @@ -121,7 +121,7 @@ def CapGains(e23250, e22250, e23660, _sep, _feided, FEI_ec_c,
> >  def SSBenefits(SSIND, MARS, e02500, _ymod, e02400, SS_thd50, SS_thd85,
> >                 SS_percentage1, SS_percentage2):
> >
> > -    if SSIND != 0 or MARS == 3 or MARS == 6:
> > +    if SSIND == 2 or MARS == 3 or MARS == 6:
>
> On Thu, 1 Oct 2015, Sean.Wang wrote: In taxcalc/functions.py: > @@ -121,7
> +121,7 @@ def CapGains(e23250, e22250, e23660, _sep, _feided, FE I_ec_c, >
> def SSBenefits(SSIND, MARS, e02500, _ymod, e02400, SS_thd50, SS_thd85, >
> SS_percentage1, SS_percentage2): > > - if SSIND != 0 or MARS == 3 or MARS
> == 6: > + if SSIND == 2 or MARS == 3 or MARS == 6: Not sure what does SSIND
> mean.
> There are a few people exempt from taxation of social security benefits
> and SSIND eq 2 indicates such people. For married-separate returns, we
> can't do the correct calculation - I forgot why. I can investigate if
> necessary. dan
> … <#15024f83cdefc428_>
> — Reply to this email directly or view it on
> GitHub.[AHvQVV7CZD2uSM1gxvEjPX6bFmiXXRM_ks5o3YNegaJpZM4GHeOp.gif]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourcePolicyCenter/Tax-Calculator/pull/370/files#r40958429>
> .
>



-- 
Zhiyuan Wang
University of Michigan, Ann Arbor Class of 2015
B.S. | *Mathematics of Finance & Risk Management*
        | *Honors in Mathematics *
        | *Statistics*
Cell: (734) 834-4232
Email: zhwa@umich.edu